UI/UnoZIndex/UnoZIndex/UnoZIndex/UnoZIndex/Presentation/MainPage.xaml

```diff
@@ -1,15 +1,28 @@
-<Page x:Class="UnoZIndex.Presentation.MainPage"
-	  xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-	  xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-	  xmlns:local="using:UnoZIndex.Presentation"
-	  xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
-	  xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
-	  mc:Ignorable="d"
-	  xmlns:uen="using:Uno.Extensions.Navigation.UI"
-	  xmlns:utu="using:Uno.Toolkit.UI"
-	  xmlns:toolkit="using:Uno.UI.Toolkit"
-	  NavigationCacheMode="Required"
-	  Background="{ThemeResource BackgroundBrush}">
+<Page
+	x:Class="UnoZIndex.Presentation.MainPage"
+	xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	xmlns:local="using:UnoZIndex.Presentation"
+	xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	xmlns:toolkit="using:Uno.UI.Toolkit"
+	xmlns:uen="using:Uno.Extensions.Navigation.UI"
+	xmlns:utu="using:Uno.Toolkit.UI"
+	Background="{ThemeResource BackgroundBrush}"
+	NavigationCacheMode="Required"
+	mc:Ignorable="d">
+
+	<Page.Resources>
+		<utu:ShadowCollection x:Key="InnerShadows">
+			<utu:Shadow
+				BlurRadius="5"
+				IsInner="False"
+				OffsetX="-5"
+				OffsetY="5"
```

Comment:
Maybe let's remove these so the cards look better when they are overlayed:

Without:
![image](https://github.com/unoplatform/Uno.Samples/assets/4793020/6e670bd9-cf17-4852-9b9a-db2d641b8495)


With:
![image](https://github.com/unoplatform/Uno.Samples/assets/4793020/d42a658b-bb6e-4452-b1a6-26e827356212)

Reply:
I am not sure I understand what you mean: This is what shows up currently 

<img width="359" alt="image" src="https://github.com/unoplatform/Uno.Samples/assets/5883457/84abe90e-d482-4641-b711-6cce0af2ded3">

Reply:
@Onotseike  I believe his suggestion was to remove the offset.
```suggestion
```

Removing it the result will be:
![image](https://github.com/unoplatform/Uno.Samples/assets/20712372/4ca2c83c-1963-4742-b6f3-637eefb958b7)

Reply:
Yeah otherwise it seems hard to see the top and right edges of the cards

Reply:
Done